nyansat/host/shell/antenny_client.py

```diff
@@ -59,6 +59,9 @@ def wrapper(*args, **kwargs):
         except ValueError as e:
             logging.error(e)
             print("Incorrect parameter type.")
+        except NoSuchFileError as e:
+            logging.error(e)
+            print("No such file")
```

Comment:
I would suggest this error be more specific. Where is the file? On the host, the station?